### DISCUSSION TITLE
Cross-check WW sample name, that you used

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#24] [Comments_HIG_23_001] [answered]
In the below table you took `WW to 2l2nu` sample. Why?

I checked the das and I can see two samples, and we should use one of them:

```bash
/WW_TuneCP5_13TeV-pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v1/MINIAODSIM


/WWTo1L1Nu2Q_4f_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2/MINIAODSIM
```

![स्क्रीनशॉट 2023-06-30, 13 46 06 पर](https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/7773dd13-fb43-4908-b396-6e0ccece5d1a)

**Comment by jialin-guo1** [comment, marked as answer]
I am using 
```shell
/WWTo2L2Nu_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2/MINIAODSIM
``` 

because our background should is that process who have 2 leptons

**Reply by ram1123** [reply]
Yes. Along with two leptons, we need two jets instead of 2 neutrinos. As you already submitted jobs on the 2l2q sample so lets wait for its outcome.

Then prepare a summary cut flow table for each background, signal and data-sideband. We will add this to the AN.

**Comment by ram1123** [comment]
Moved to the discussion: https://github.com/ram1123/2l2Q_limitSettingTool/discussions/25
